cmd/fyne/internal/commands/build.go

```diff
@@ -331,6 +332,12 @@ func injectMetadataIfPossible(runner runner, srcdir string, app *appData,
 		return nil, nil
 	}
 
+	fyneGoModVersionConstraintOlderThan203 := version.NewConstrainGroupFromString(">=2.3")
+	if fyneGoModVersionConstraintOlderThan203.Match(fyneGoModVersionNormalized) {
+		app.VersionOlderThan203 = true
+	}
+	log.Println("fyne version:", fyneGoModVersionNormalized)
```

Comment:
I don't think this log is needed

Reply:
Oh, I was thinking I did remove it, but seems not. Good catch!